docs/src/user-docs/guides-using-presto.md

```diff
@@ -115,7 +148,7 @@ Using Presto with CLP requires:
 5. Start a Presto cluster by running:
 
     ```bash
-    docker compose up
+    docker compose up --detach
```

Comment:
it might be better to also add the `--wait` flag so users can be notified if any service fails to start successfully

@coderabbitai add an issue to track